prow/config.yaml

```diff
@@ -6001,29 +5562,61 @@ presubmits:
       containers:
       - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
         args:
+        - "--job=$(JOB_NAME)"
         - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
         - "--root=/go/src"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - "--clean"
+        - --service-account=/etc/service-account/service-account.json
```

Comment:
is this intended?

Reply:
--clean was not though shouldn't hurt since no cache volume now. I'll rever that.